packages/migrate/transforms/middleware.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable consistent-return */
+const camelcase = require('camelcase');
+
+const getIdentifierFromPackage = (pkg) => {
+  const parts = pkg.split('/');
+
+  return camelcase(parts[parts.length - 1]);
+};
+
+/**
+ * Transform .neutrinorc.js files to remove legacy middleware string-based
+ * formats in module.exports.use and neutrino.use(). Will move string formats
+ * as require()s to top of file.
+ * use: ['@neutrinojs/react'] -> use: [react()]
+ * use: [['@neutrinojs/react', { options }]] -> use: [react({ options })]
+ * neutrino.use('@neutrinojs/react') -> neutrino.use(react())
+ * neutrino.use('@neutrinojs/react', options) -> neutrino.use(react(options))
+ */
+module.exports = ({ source }, { jscodeshift: j }) => {
```

Comment:
When I tried migrate against Treeherder's `.neutrinorc.js`, it left these middleware usages unchanged:
https://github.com/mozilla/treeherder/blob/4c01ee963b1f227101885864f881ccf3918561eb/.neutrinorc.js#L54-L56
https://github.com/mozilla/treeherder/blob/4c01ee963b1f227101885864f881ccf3918561eb/.neutrinorc.js#L130-L132

This style of usage is likely not that common, but thought I'd mention in case it would be worth seeing if it was easy to support without false positives. (They would probably also want to be written using inline `require()` to keep startup time low if the conditional is not true.)